### PR TITLE
Added new server code to avoid endpoint mix-up

### PR DESCRIPTION
**Pull request by anmolshl**
There is a new server class that separates out the information endpoint from the rest of the path finding endpoints. @palango

### CODE REVIEW

File: pathfinder/metrics_server.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 import logging
 import sys
 from typing import List
+import json
```
Comment (palango) [inline pathfinder/metrics_server.py]: why is this needed?

Comment (anmolshl) [reply pathfinder/metrics_server.py]: Ahh I was supposed to remove that!

**Comment by anmolshl** [comment]
@palango updated!
